packages/subiquity_client/lib/src/types.dart

```diff
@@ -269,12 +269,13 @@ class WSLConfigurationAdvanced with _$WSLConfigurationAdvanced {
   const factory WSLConfigurationAdvanced({
```

Comment:
same here, we could use `@JsonSerializable(fieldRename: FieldRename.snake)` to avoid the error-prone manual field name mapping.